### ISSUE TITLE
Autosizing Heat Recovery Water Flow Rate with Chiller:Electric:EIR

### ISSUE THREAD

**Issue by asparke2** [NREL/EnergyPlus#5764]
The IDD for  `Chiller:Electric:EIR` shows `Design Heat Recovery Water Flow Rate` as autosizable.

```
  N14, \field Design Heat Recovery Water Flow Rate
       \type real
       \note If non-zero, then the heat recovery inlet and outlet node names must be entered.
       \note Heat recovery is only available with Condenser Type = WaterCooled.
       \units m3/s
       \minimum 0.0
       \default 0.0
       \autosizable
       \ip-units gal/min
```

However, as Chandan Sharma pointed out in response to [this UnmetHours question](https://unmethours.com/question/18569/heat-recovery-chiller-autosizing/), the autosizing code appears to be missing.  @Myoldmopar pointed out that in place of the autosizing code there is [a very helpful comment](https://github.com/NREL/EnergyPlus/blob/caa0dd3339009d9a572ac23fab6b6ebfac777055/src/EnergyPlus/ChillerElectricEIR.cc#L1363).

Ideally this would be fixed by adding the autosizing code instead of removing 'autosizable' from this field in the IDD.

**Comment by EnergyArchmage** [comment]
Planning on addressing this as part of a separate project to fill various autosizing gaps in plant equipment including HR sizing for two other chillers.

**Comment by mbadams5** [comment]
@EnergyArchmage Was this addressed by #5740? If so, can this be closed?

**Comment by EnergyArchmage** [comment]
@mbadams5 , yes it was. 

code is here
https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/ChillerElectricEIR.cc#L1381
